### ISSUE TITLE
`trpcServer.query.ssr()` always returns undefined

### ISSUE THREAD

**Issue by ferntheplant** [ottomated/trpc-svelte-query#7]
I'm loading a route like `/dashboard/[itemId]` and am using `trpcServer.getItemById.ssr(...)` to prefetch the item before using it on the client side. However I want detect if the item with this id does not exist by doing something like

```typescript
const item = await trpcServer.getItemById.ssr(event.params.itemId, event)
if (!item) {
    throw error(404, 'No item found!')
}
```

The `.ssr()` function signature suggests it returns the data of the query but it seems to always return `undefined`. Is this is a bug or is there a better way to accomplish this?

**Comment by ottomated** [comment]
This might be a bug, but if you throw a TRPCError with code NOT_FOUND from your query that will propagate to sveltekit.

**Comment by ottomated** [comment]
Fixed in 1.0.2

**Comment by ferntheplant** [comment]
thanks!
